moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -717,13 +717,6 @@ const ProjectFundingTable = () => {
             updateProjectFundingData.funding_amount = Number(
               newData.funding_amount
             );
```

Comment:
we do this in the update row, but not in the add new row. So when a user updates a row that didnt have any funding amount associated with it, the activity log registers it as being updated (null to a number)

Reply:
ah hah. can we just save a `null` value in the db instead of `0`? this seems like non-normal behavior to me.